src/libs/notifications.ts

```diff
@@ -118,18 +145,6 @@ const NotificationDisplay = ({ id }) => {
                 },
                 ['Details']
               ),
-            !!action &&
```

Comment:
removing support for action mechanic since it does not appear to be used by any consumers.  This need appears to be served by passing in rich ReactNode content through optional NotificationsProps.message arg instead.